packages/reference/src/resources/ExternalResourceCard/ExternalResourceCard.tsx

```diff
@@ -27,10 +38,55 @@ const defaultProps = {
   hasCardRemoveActions: true,
 };
 
+const styles = {
+  subtitle: css({
+    color: tokens.gray600,
+    marginBottom: 'none',
+  }),
+  description: css({
+    color: tokens.gray900,
+    marginBottom: 'none',
```

Comment:
is that legal? I thought it was a Forma feature not css

Reply:
Doesn't it get translated to that same css property? 🤔

Reply:
no, afaik it does not.